src/content/datachannel/messaging/main.js

```diff
@@ -0,0 +1,168 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+'use strict';
+import {LitElement, html} from 'https://unpkg.com/@polymer/lit-element?module';
+
+class MessagingSample extends LitElement {
+  constructor() {
+    super();
+    this._localConnection = this._remoteConnection = this._localChannel = this._remoteChannel = null;
+  }
+
+  disconnect(e) {
+    this._root.querySelector('#disconnectButton').disabled = true;
+    this._localConnection.close();
+    this._remoteConnection.close();
+  }
+
+  async connect(e) {
+    try {
+      this._root.querySelector('#connectButton').disabled = true;
+      const dataChannelParams = {ordered: false};
+      let lpc = this._localConnection = new RTCPeerConnection(null);
```

Comment:
new RTCPeerConnection()
-- passing null was just to workaround Chrome being picky about the lack of arguments in Chrome < 51